websites/F/FiveM/presence.ts

```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+  clientId: "730897382937591848"
+});
+
+const browsingTimestamp = Math.floor(Date.now() / 1000),
```

Comment:
merge these consts and then codacy should also update.

Reply:
resolved